ibis/pandas/tests/test_operations.py

```diff
@@ -1446,3 +1446,12 @@ def test_scalar_broadcasting(batting, batting_df):
     result = expr.execute()
     expected = batting_df.assign(demeaned=batting_df.G - batting_df.G.mean())
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize('raw_value', [0.0, 1.0])
+def test_scalar_parameter(t, df, raw_value):
+    value = ibis.param(dt.double)
+    expr = t.float64_with_zeros == value
+    result = expr.execute(params={value: raw_value})
```

Comment:
I think we can support both. I *think* we can implement the named version as a special case of unnamed. Implemented the dict API version first because it's more general since it doesn't care about the name of the value.